docker-java/src/test/java/com/github/dockerjava/cmd/ListImagesCmdIT.java

```diff
@@ -54,6 +55,13 @@ public void listImagesWithDanglingFilter() throws DockerException {
         assertTrue(imageInFilteredList);
     }
 
+    @Test
+    public void listImagesWithReferenceFilter() throws DockerException {
+        List<Image> images = dockerRule.getClient().listImagesCmd().withReferenceFilter("busybox")
+            .exec();
+        assertThat(images, hasSize(1));
```

Comment:
we cannot assume that local Docker will only have one `busybox` image. Actually, CI is failing due to that :)

Consider tagging an image (see other tests) with a unique identifier and checking for it instead.